src/openrct2-dll/openrct2-dll.cpp

```diff
@@ -41,7 +41,9 @@ static char * * GetCommandLineArgs(int argc, wchar_t * * argvW);
 static void FreeCommandLineArgs(int argc, char * * argv);
 static char * ConvertUTF16toUTF8(const wchar_t * src);
 
-DLLEXPORT int LaunchOpenRCT2(int argc, wchar_t * * argvW)
+DLLEXPORT int LaunchOpenRCT2(int argc, wchar_t ** argvW);
```

Comment:
Why is this line here?

Reply:
Was getting warnings about missing decls, but I'm not sure if this is still applicable...